apps/projects/app/components/Content/Issues.js

```diff
@@ -228,10 +228,19 @@ class Issues extends React.PureComponent {
     </StyledIssues>
   )
 
+  getExpLevels = () => {
+    let expLevels = []
```

Comment:
we should declare every variable as `const` unless we have a good reason to do it, in this case both `expLevels` and `a` can be declared as `const` variables (you can still mutate `expLevels` by pushing new values, `const` just prevent reassignments)

Reply:
Acknowledged. ExpLevels are moving, I'll keep the above in mind :)